tests/runtimes/simple-keyvalue/Cargo.toml

```diff
@@ -17,7 +17,7 @@ path = "src/upgraded.rs"
 [package.metadata.fortanix-sgx]
 heap-size = 134217728
 stack-size = 2097152
-threads = 8
+threads = 74
```

Comment:
These threads are bind to CPU cores right, e.g. only cpu-cores number of enclave threads can run at the same time? So it might be be good to update the "recommended" specs to a bit more than 4 in the documentation once this is live, as it likely will be beneficial?